hedera-node/hedera-app/src/main/java/com/hedera/node/app/grpc/impl/MethodBase.java

```diff
@@ -185,7 +185,7 @@ public void invoke(
             @NonNull final Metrics metrics,
             @NonNull final String nameTemplate,
             @NonNull final String descriptionTemplate) {
-        final var baseName = serviceName + "/" + methodName;
+        final var baseName = serviceName + ":" + methodName;
```

Comment:
Also don't we need to change lines 
```
private static final String SPEEDOMETER_HANDLED_NAME_TPL = "%sHdl/sec";
private static final String SPEEDOMETER_RECEIVED_NAME_TPL = "%sRcv/sec";
```

Reply:
I think you are looking at outdated code. This has already been fixed recently.

Reply:
Ah yes! thanks